### ISSUE TITLE
The installation stucks at Prepare

### ISSUE THREAD

**Issue by JaneValleyhasBtaken** [nongthaihoang/custom_font_installer#6]
It stuck at the "prepare" phase.
It is the newest Magisk Alpha if to be accurate

**Comment by nongthaihoang** [comment]
I'm not sure what Magisk Alpha is. I'm already on the latest Magisk Stable v26.4. I'm only making sure the module is working fine for this version.

**Comment by BloodRaven0** [comment]
@nongthaihoang  I'm having the same problem on Magisk Stable v26.4 (26400).
I installed OMF with AFDKO extension successfully, but the installation of CMF still gets stuck at +Prepare.
I'm on Android 10 AOSP-based ROM if that's relevant.

I even tried to install an older version (CFI_2.3.zip) that I have installed in the past successfully on this phone, but it returned an error:
! /sbin/mirror/system/etc/fonts.xml: file not found
! Installation failed

As a whole the installation of this module is very confusing, I couldn't find a comprehensive step-by-step guide on installing it with all the prerequisites listed. Is there such a documentation anywhere?

**Comment by nongthaihoang** [comment]
Send the Magisk Installation Log.

**Comment by BloodRaven0** [comment]
[magisk_install_log_2024-01-04T22.59.54.log](https://github.com/nongthaihoang/custom_font_installer/files/13854149/magisk_install_log_2024-01-04T22.59.54.log)
[magisk_install_log_2024-01-07T18.20.12.log](https://github.com/nongthaihoang/custom_font_installer/files/13854150/magisk_install_log_2024-01-07T18.20.12.log)

@nongthaihoang  Here are two logs of trying to install the latest version.

**Comment by nongthaihoang** [comment]
The logs say the module couldn't find the system fonts.xml. What's the output of this  command?
╰─❯ ls -la $(magisk --path)

**Comment by BloodRaven0** [comment]
> The logs say the module couldn't find the system fonts.xml. What's the output of this command? ls -la $(magisk --path)

![Screenshot_20240108-123422_Termux](https://github.com/nongthaihoang/custom_font_installer/assets/42444745/b09fea59-14f3-4524-8c58-0eebabdba88c)
Here it is.
So the problem is essentially the same as the 2.3 version of the module was having - not being able to find the fonts.xml file?
That is strange, since as I said, I have installed it successfully in the past on this rom (Resurrection remix 8.6.9). Maybe something went wrong when uninstalling the module?

I don't kniw if it's relevant to this issue, but I did manage to successfully install another module that changes the font after running the termux command as instructed in its readme: https://github.com/charityrolfson433/mffmv11/tree/main

Let me know if I can do anything further that can help diagnose the problem. :)
